### PR TITLE
Build Script for drupal-ultimate_cron-v2.0

### PR DESCRIPTION
**Pull request by HarithaPatchari054**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: d/drupal-ultimate_cron/drupal-ultimate_cron_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+#   
+# -----------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=ultimate_cron
+CORE_PACKAGE_NAME=drupal
+PACKAGE_URL=https://git.drupalcode.org/project/ultimate_cron
+CORE_PACKAGE_URL=https://github.com/drupal/drupal
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-7.x-2.0}
```
Comment (kumarvikas2020) [inline d/drupal-ultimate_cron/drupal-ultimate_cron_ubi_8.5.sh:27]: The version here is not matching with the version at line no 5.

**Review by kumarvikas2020 — CHANGES_REQUESTED** [review]

File: d/drupal-ultimate_cron/drupal-ultimate_cron_ubi_8.5.sh (hunk 1 above)
Comment (HarithaPatchari054) [reply d/drupal-ultimate_cron/drupal-ultimate_cron_ubi_8.5.sh:27]: @kumarvikas2020 Made changes to match version

**Comment by prashantkhoje** [comment]
LGTM

**Review by kumarvikas2020 — APPROVED** [review]

**Comment by kumarvikas2020** [comment]
@seth-priya LGTM

**Pull request merged**